website/components/LandingPage/index.tsx

```diff
@@ -323,7 +324,7 @@ export default function LandingPage(): ReactElement {
       <Banner>
         <BannerVideoContainer style={{ zIndex: 1 }}>
           <video autoPlay={allowMotion} loop muted>
-            <source src="/videos/banner-video.mp4" type="video/mp4" />
+            <source src={BannerVideo} type="video/mp4" />
```

Comment:
is this line still downloading the banner video or is it inlining it as something like a data url?

Reply:
It'll link to the banner video file! Doing it like this lets webpack configure the final path to the video asset.